src/VirtoCommerce.Platform.Security/Services/CustomUserManager.cs

```diff
@@ -140,6 +143,13 @@ public override async Task<IdentityResult> UpdateAsync(ApplicationUser user)
             //we must expire user before update, becouse if we get user from cache,
             //then we get EF exception (can't trak entity with same id)
             SecurityCacheRegion.ExpireUser(user);
+
+            var currentTime = DateTime.UtcNow;
```

Comment:
look at ApplicationBuilderExtensions.UseDbTriggers
this is redundant code

Reply:
I know. It's temporary. Unfortunately, Triggers<IAuditable> does not catching ApplicationUser.

Reply:
have to fix it

Reply:
Fixed.